packages/connection/src/class-manager.php

```diff
@@ -26,7 +26,14 @@ class Manager {
 	const SECRETS_EXPIRED        = 'secrets_expired';
 	const SECRETS_OPTION_NAME    = 'jetpack_secrets';
 	const MAGIC_NORMAL_TOKEN_KEY = ';normal;';
-	const JETPACK_MASTER_USER    = true;
+
+	/**
+	 * For internal use only. If you need to get the connection owner, use the provided methods
+	 * get_connection_owner_id, get_connection_owner and is_get_connection_owner
+	 *
+	 * @var boolean
+	 */
+	const JETPACK_MASTER_USER = true; //phpcs:ignore ..Jetpack_Sniffs_MasterUserConstant.ShouldNotBeUsed
```

Comment:
Could we rename this to something else? Or just pass true through these functions? I think it would be less confusing if we stopped using this constant name completely.

Reply:
sure! it's a good idea, it actually crossed my mind at some point

Reply:
done!

Reply:
I don’t think we can safely rename this. We need to maintain backward compatibility with consumer plugins, and WCPay uses `Manager::JETPACK_MASTER_USER` [here](https://github.com/Automattic/woocommerce-payments/blob/master/includes/wc-payment-api/class-wc-payments-http.php#L59). 

Also, I think there’s at least one version of Jetpack that uses `Manager::JETPACK_MASTER_USER` in the Sync package. That version of Jetpack would not be compatible with this version of the `jetpack-connection` package.

Reply:
> I don’t think we can safely rename this. We need to maintain backward compatibility with consumer plugins, and WCPay uses Manager::JETPACK_MASTER_USER

I suppose we can deprecate it for now and then remove it in a new version of the package eventually once they've updated to the new method.

But understood on not removing it immediately.